charts/tekton-apps/Chart.yaml

```diff
@@ -13,7 +13,7 @@ triggersVersions: "v0.16.0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.15
+version: 0.1.16
```

Comment:
Pls add docs below outlining HOW to use wordpress. Sample.

Reply:
done